### PR TITLE
Adding support for LDFLAGS and SLDFLAGS to pickup value from environment

### PR DESCRIPTION
**Pull request by lkammath**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -566,6 +566,14 @@
 # If the user passed thru extra defines, grab them
 if "DEFINES" in os.environ:
     DEFINES = os.environ["DEFINES"]
+# linker flags - LDFLAGS, SLDFLAGS
+if ("LDFLAGS" in os.environ):
+    LDFLAGS = os.environ["LDFLAGS"]
+if ("SLDFLAGS" in os.environ):
+    SLDFLAGS = os.environ["SLDFLAGS"]
+
+# Take the ones from the environment or command line,
```
Comment (janssens2) [inline config.py]: comments should be prior to what you are doing, would recommend putting the comments before line 570

Comment (janssens2) [reply config.py]: This comment seems duplicated from earlier in the code, line 560.  These comment lines here are unnecessary.

Comment (lkammath) [reply config.py]: sorry. it was copy paste mistake

**Comment by lkammath** [comment]
Can this be merged?

**Comment by kahnevan** [comment]
Please create a pull request for this on the ecmd15 branch @lkammath .

**Comment by lkammath** [comment]
Pushed into the ecmd15 branch. please take look

https://github.com/open-power/eCMD/pull/295/

**Pull request merged**